content/payments/billpay/pre-built-screens/quickstart/iOS.mdx

```diff
@@ -73,61 +73,80 @@ The above request will return URL based on the input params. Here’s a sample
 
 ##### Switch control from your app to Setu
 
-The link returned by the API should be loaded within `WKWebView` in a `ViewController` which implements `UIViewController`, and `WKScriptMessageHandler`. The controller must also extend and use a custom `WKNavigationDelegate` class to intercept and download PDF files.
+The link returned by the API should be loaded within a `WKWebView`, facilitated by SwiftUI's `WebView`. `ContentView` controls the main view, which incorporates navigation controls and a custom back button. `WebViewCoordinator` manages navigation actions and delegates for smooth interaction with web content.
 
 If you’re using a cross platform solution—like Flutter, React Native, Ionic—look at the code snippets provided <a href="/payments/billpay/quickstart/cross-platform" target="_blank">here</a>.
 
-The `userContentController` will need to implement one function—
-
--   `unload` — Used by the parent app to dismiss the webview
-
-<Callout type="tip">
-    <b>Special use case:</b> This <code>unload</code> function can also be used for dismissing the webview and redirecting a user back to your native app once a bill payment journey is completed (i.e. payment is successful) via a CTA from the Setu webview. Please let our team know if you would like to enable this use case for your app.
-</Callout>
-<br />
-
-<CodeBlockWithCopy language="bash">
-{`// The iOS observer function
-func userContentController(_ userContentController: WKUserContentController, didReceive message: WKScriptMessage) {
-    guard let dict = message.body as ? [String: AnyObject]
-    else {
-        return
+Here’s a sample code for the same -
+
+<CodeBlockWithCopy language="jsx">
```

Comment:
I know it doesn't matter for this case,  but let's set the language properly

Reply:
It was not working with swift, that's why set it to different, it doesn't prettify it.